test/TestOpTensorCreate.cpp

```diff
@@ -113,3 +113,21 @@ TEST(TestOpTensorCreate, NoErrorIfTensorFreedBefore)
     EXPECT_FALSE(tensorA->isInit());
     EXPECT_FALSE(tensorB->isInit());
 }
+
+
+TEST(TestOpTensorCreate, ExceptionOnZeroSizeTensor)
+{
+    std::vector<float> testVecA;
+
+    std::shared_ptr<kp::Tensor> tensorA{ new kp::Tensor(testVecA) };
+
+    kp::Manager mgr;
+
+    try{
+        mgr.evalOpDefault<kp::OpTensorCreate>({ tensorA });
+    } catch( const std::runtime_error& err ) {
+         // check exception
+        ASSERT_STREQ("Kompute Tensor attempted to create a zero-sized buffer", err.what() );
```

Comment:
StrEq comparison may be too brittle for test, eventually it would be interesting to explore structured errror/exception handling, but for now is there a way to just have a check for "zero sized" in the string?

Reply:
May also be worth adding an extra comment in the constructor so it's explicit that it will throw an error if empty tensor